app/views/admin/kudos/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2>Admin Kudos List</h2>
```

Comment:
As above consider renaming it to `Kudos` List

Reply:
True! Will fix.